### PR TITLE
Pool overview

### PR DESCRIPTION
**Pull request by shan-57blocks**
See: https://github.com/00labs/huma-dapp/pull/582

### CODE REVIEW

File: packages/huma-shared/src/v2/utils/poolConfigContract.ts [hunk 1]
```diff
@@ -0,0 +1,168 @@
...
+import { FirstLossCoverConfigStructOutput } from '../abis/types/PoolConfig'
+import { FirstLossCoverIndex } from '../types'
+import { BP_FACTOR } from './const'
+import { PoolInfoV2 } from './pool'
+import { getPoolConfigContractV2, getPoolInfoV2 } from './poolContract'
+
+const getProtocolFeeInBps = async (
+  poolConfigContract: PoolConfig,
+  provider: JsonRpcProvider | Web3Provider | undefined,
+) => {
```
Comment (mliu) [inline packages/huma-shared/src/v2/utils/poolConfigContract.ts:20]: Can we add return type hints?

File: packages/huma-shared/src/v2/utils/poolConfigContract.ts [hunk 2]
```diff
@@ -0,0 +1,168 @@
...
+    HUMA_CONFIG_ABI,
+    provider,
+  ) as HumaConfig
+  return humaConfigContract.protocolFeeInBps()
+}
+
+const getFlcConfigs = async (
+  poolInfo: PoolInfoV2,
+  poolConfigContract: PoolConfig,
+) => {
```
Comment (mliu) [inline packages/huma-shared/src/v2/utils/poolConfigContract.ts:33]: Ditto

File: packages/huma-shared/src/v2/utils/poolConfigContract.ts [hunk 3]
```diff
@@ -0,0 +1,168 @@
...
+    Object.values(poolInfo.firstLossCovers).map((firstLossCover: string) =>
+      poolConfigContract.getFirstLossCoverConfig(firstLossCover),
+    ),
+  )
+  return flcConfigs
+}
+
+const getFlcTotalAssetsAfterRisk = (
+  flcConfigs: FirstLossCoverConfigStructOutput[],
+) => {
```
Comment (mliu) [inline packages/huma-shared/src/v2/utils/poolConfigContract.ts:44]: Ditto

**Review by mliu — APPROVED** [review]

File: packages/huma-shared/src/v2/utils/poolConfigContract.ts (hunk 1 above)
Comment (shan-57blocks) [reply packages/huma-shared/src/v2/utils/poolConfigContract.ts]: Updated

File: packages/huma-shared/src/v2/utils/poolConfigContract.ts (hunk 2 above)
Comment (shan-57blocks) [reply packages/huma-shared/src/v2/utils/poolConfigContract.ts]: Updated

File: packages/huma-shared/src/v2/utils/poolConfigContract.ts (hunk 3 above)
Comment (shan-57blocks) [reply packages/huma-shared/src/v2/utils/poolConfigContract.ts]: Updated

**Pull request merged**